app/controllers/application_controller.rb

```diff
@@ -1,2 +1,21 @@
 class ApplicationController < ActionController::API
+  attr_reader :current_user
+
+  before_action :authenticate_user!
+
+  private
+
+  def authenticate_user!
+    authentication = RecognizeUser.call(token: auth_token)
+
+    if authentication.success?
+      @current_user = authentication.user
+    else
+      render json: { error: authentication.error }, status: :unauthorized
```

Comment:
I do not see a translation for `"error.invalid_jwt"` in the locales. I think we can use this code here:

```suggestion
      render json: { error: "unauthorized" }, status: :unauthorized
```

And I think we should return this error if the user is `nil`.